modules/taskFunctions.js

```diff
@@ -0,0 +1,61 @@
+class TaskCollection {
+  constructor() {
+    this.listContainer = document.querySelector('.tasks-container');
+    this.description = document.querySelector('#addDo');
+    this.completed = false;
+    this.taskIndex = 1;
+    this.toDoList = [];
+  }
+
+  storedLocal() {
+    if (localStorage.getItem('tasks') == null) {
+      localStorage.setItem('tasks', JSON.stringify(this.toDoList));
+    }
+
+    const store = localStorage.getItem('tasks');
+    if (store) {
+      this.toDoList = JSON.parse(store);
+    }
+  }
+
+  removeTask(index) {
```

Comment:
One more suggestion I have for you is to update the indexes of the remaining tasks after deleting a task. This would ensure that the indexes represent the current list order and are unique. 🗑️

To implement this change, you could follow these steps:

- In the removeTask method, after filtering out the deleted task from the toDoList array, use a for loop to iterate over the remaining tasks and assign them a new index based on their position in the array.
- In the displayTasks method, use task.index instead of task.taskId as the id for the task items.

This would make your code more robust and avoid errors. 😊

Reply:
Thank you for the review. on this line, please note the following: 

- I have the for loop iterating the remaining task and assigning a new index.
- In the displayTasks method: I do not have task.taskId used in it

Reply:
> Thank you for the review. on this line, please note the following:
> 
> * I have the for loop iterating the remaining task and assigning a new index.
> * In the displayTasks method: I do not have task.taskId used in it

@sulaimonaa, Thank you very much for your question. Noted. 
My suggestion was that you ensure that you delete a task, and the index of all tasks should be updated so if I delete task one and I have two tasks, the index of task two then changes to 1.

https://www.loom.com/share/98dea1ad3b29488a9c55309876ea49cd?sid=3489369e-6801-4acb-b6a6-90337375e871